### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by domikoz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,381 +1,246 @@
...
 &nbsp;
 
 <div align="center">
-    <img src="https://neptune.ai/wp-content/uploads/Neptune-customers.png" width="900" />
+    <img src="https://neptune.ai/wp-content/uploads/2022/11/github-customers.png" width="900" />
 </div>
 &nbsp;
 
 Read how various customers use Neptune to [improve their workflow](https://neptune.ai/customers).
+&nbsp;
```
Comment (Raalsky) [inline README.md:229]: Why do we need to add HTML-like whitespace? Does Markdown doesn't allow multiple newlines?

Comment (domikoz) [reply README.md:229]: Should I use <br> </br>?

File: README.md [hunk 2]
```diff
@@ -1,381 +1,242 @@
...
-    <a href="https://twitter.com/neptune_ai"><img src="https://img.shields.io/twitter/follow/neptune_ai.svg?label=Follow" alt="twitter"></a>
-    <a href="https://www.youtube.com/channel/UCvOJU-ubyUqxGSDRN7xK4Ng"><img src="https://img.shields.io/youtube/views/9iX6DxcijO8?style=social" alt="youtube"></a>
+  <a href=https://docs.neptune.ai/getting-started/hello-world target=“_blank”>Quickstart</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://neptune.ai/ target=“_blank”>Website</a>
```
Comment (Raalsky) [inline README.md:10]: Something went wrong I think (missing doublequotes). It should be: `<a href="https://neptune.ai/" target=“_blank”>Website</a>`

Comment (Raalsky) [reply README.md:10]: Probably that's a reason why it's still not opening in a new tab (according to `target="_blank"). This affects most of the URLs here.

File: README.md [hunk 3]
```diff
@@ -1,381 +1,242 @@
...
-    "batch_size": 64,
-    "dropout": 0.2,
-    "optim": {"learning_rate": 0.001, "optimizer": "Adam"},
-}
-
-for epoch in range(100):
-    run["train/accuracy"].log(epoch * 0.6)
-    run["train/loss"].log(epoch * 0.4)
+run = neptune.init_run("Me/MyProject")
+run["parameters"] = {"lr":0.1,
```
Comment (Raalsky) [inline README.md:63]: `black` should be applied or just reformat this particular snippet. There is a lack of whitespaces before operations.

Comment (domikoz) [reply README.md:63]: Not sure if I get that? How I can reformat it or apply `black`?

File: README.md (hunk 2 above)
Comment (Raalsky) [reply README.md]: The double quotes around `_blank` are weird. Should be `"` vs `“`

File: README.md (hunk 3 above)
Comment (Raalsky) [reply README.md:63]: Ohh. Just replace it with `{"lr": 0.1, "dropout": 0.4}` (spaces before every number). https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ACtAG5dAD2IimZxl1N_WmXhZA7zE3fzn9obYcWFsOiHVbW-Mluhf2KwGtf8W4pYAywlOxfyq6KX__um_7gmFtjG8I9ZDxnUWYNFs61M8-6xW_ZWBTF2RsLwv3yzLIvl3ox0O12rimPLh1ErKyJk2LIsINwAAAAAqlub-mwJnt8AAYoBrgEAANLOaO2xxGf7AgAAAAAEWVo=

Comment (domikoz) [reply README.md]: thanks, hope it's ok now :)

File: README.md (hunk 2 above)
Comment (domikoz) [reply README.md]: changed, but I see that it's an issue on github side, will delete it then

Comment (Raalsky) [reply README.md]: Ok

**Review by Raalsky — APPROVED** [review]
